var/spack/repos/builtin/packages/cpmd/package.py

```diff
@@ -97,4 +97,8 @@ def test(self):
             exe_name = 'cpmd.x'
         opts.append(test_file)
         opts.append(test_dir)
-        self.run_test(exe_name, options=opts)
+        expected = ['2       1        H        O              1.84444     0.97604',
+                    '3       1        H        O              1.84444     0.97604',
+                    '2   1   3         H     O     H              103.8663'
```

Comment:
Just my two cents (not a request), but I always :cold_sweat: when I see floating points used as expected output. Is there any tolerance taken into account or is it just string comparison?

Reply:
This is just string comparison.
The test was OK in x86%gcc, a64fx%gcc, and a64fx%fj.

Reply:
Could you see my replies?

Reply:
@ketsubouchi Sure, I was just waiting for somebody to chime in - the one above was just a comment, not a request for change. Anyhow, since nobody else commented, and tests are not mandatory I'll merge (even though I consider tests done on literal floating points to be very fragile).